app_flutter/README.md

```diff
@@ -6,4 +6,51 @@
 
 `flutter run -d chrome --web-port=8080`
 
-Must run on port 8080 for Google Sign In to work since that is the only enabled port for localhost.
+Must run on port 8080 for Google Sign In to work since that is the
+only enabled port for localhost.
+
+## Style
+
+### File organization
+
+The `lib` directory is organized in subdirectories as follows:
+
+* `lib/`: main.dart and top-level widgets (e.g. the build dashboard
+  widget). Can import anything.
+
+* `lib/widgets/`: Custom widgets used in this project. The files in
+  this directory should import either the Flutter widgets package or
+  the Flutter material package, and may import any local files except
+  those in the top-level `lib/` directory.
+
+* `lib/state/`: States, objects that interface between widgets and the
+  services. Files in this directory should not import any Flutter
+  packages other than foundation, and should not import local files
+  that relate to UI (e.g. those in `widgets/`).
+  
+* `lib/service/`: Services, objects that interface with the network.
+  Files in this directory should not import any Flutter packages other
+  than foundation, and should not import any local files from other
+  directories.
+  
+* `lib/logic/`: Other code, mainly non-UI utility methods and data
```

Comment:
TIL utils is an unclear name :)

Reply:
well we already have a utils in the tests directory, so... :-)